DuggaSys/microservices/monolithic-to-microservices.md

```diff
@@ -0,0 +1,123 @@
+
+Disclaimer, this guide serves as a guideline and can not be followed 100% since the structure of all files differ. It's recommended to test after each step which include code changing to so its faster to spot if something goes wrong. When this guide was written no service using retrive information was used, write guide at step 5.
+***
+##Setting up file & folder:
+***
+
+###Step 1
+Locate the folder microservices:
+***DuggaSys -> microservices***
+
+###Step 2
+If a folder named after the monolithic service file exists skip to step 3.
+Else if it does not exist, create a folder named after the service file, for example codeviewerService.php(monolithic) becomes codeviewerService(Folder).
+
+###Step 3
+Create a file with name following the naming convention in Potential_micro_services:
+***Backend-models -> microservices -> duggaSys services***
+
+**Example:**
+| codeviewerService |
+| --- |
+| editBoxTitle_ms.php |
+| createBox_ms.php |
+
+***
+##Converting monolithic to microservice:
+###Step 1
+Copy the entire service file content from the monolithic php to your new made, the purpose of this is to simmer down 
```

Comment:
Wouldn't it be easier to just select the code that is needed rather than copying an entire file and then removing unnecessary parts?

Reply:
For people like you and me who understand the code yes, for people who actually will work with this, no